tests/integ/modin/series/test_astype.py

```diff
@@ -286,7 +287,7 @@ def test_astype_from_DatetimeTZDtype_to_datetime64(from_tz):
     native = native_pd.Series([0, 1, 2, 3], dtype=from_dtype)
     snow = pd.Series(native)
     expected_dtype = get_expected_dtype(to_dtype)
-    with SqlCounter(query_count=1):
+    with SqlCounter(query_count=3):
```

Comment:
Why is the increase in the query count?

Reply:
the code below called `.astype` twice which trigger the query to get tz each time.

Reply:
I could keep improving this in a follow up PR.

Reply:
actually this can be an easy fix. let me do it now.

Reply:
Much better. Thanks!